src/Geta.Optimizely.ProductFeed/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Geta Digital. All rights reserved.
+// Licensed under Apache-2.0. See the LICENSE file in the project root for more information
+
+using System;
+using Geta.Optimizely.GoogleProductFeed.Repositories;
+using Geta.Optimizely.ProductFeed.Configuration;
+using Geta.Optimizely.ProductFeed.Repositories;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Geta.Optimizely.ProductFeed
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddProductFeed(
+            this IServiceCollection services)
+        {
+            return services.AddProductFeed(_ => { });
+        }
+
+        public static IServiceCollection AddProductFeed(this IServiceCollection services, Action<ProductFeedOptions> setupAction)
+        {
+            services
+                .AddControllers()
+                .AddXmlSerializerFormatters();
```

Comment:
I think we should not call framework service registration calls. Instead we should add pre-requisites in docs.